src/sentry/api/endpoints/organization_code_mappings.py

```diff
@@ -154,7 +153,7 @@ def get(self, request, organization):
 
         # front end handles ordering
         # TODO: Add pagination
-        data = map(lambda x: serialize(x, request.user), queryset)
+        data = list(map(lambda x: serialize(x, request.user), queryset))
         return self.respond(data)
```

Comment:
Also don't understand why this is an improvement?

Reply:
This is done in an optimization pass that I linked in the description. I guess we'd prefer it to be all rolled up in 1 PR though. https://github.com/getsentry/sentry/pull/28080#issuecomment-908578834